ptpython/entry_points/run_ptpython.py

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/env python
 """
 ptpython: Interactive Python shell.
-Usage:
-    ptpython [ --vi ]
-             [ --config-dir=<directory> ] [ --interactive=<filename> ]
-             [--] [ <arg>... ]
-    ptpython -h | --help
-
-Options:
-    --vi                         : Use Vi keybindings instead of Emacs bindings.
-    --config-dir=<directory>     : Pass config directory. By default '$XDG_CONFIG_HOME/ptpython'.
-    -i, --interactive=<filename> : Start interactive shell after executing this file.
-
-Other environment variables:
-PYTHONSTARTUP: file executed on interactive startup (no default)
+
+positional arguments:
```

Comment:
Can you start these section names with a capital?

Reply:
This is output from `argparse` (so the extra section I added I kept the same case).

Happy to do whatever, but yeah unless you overload argparse's output it's lowercase when actually emitted.

Reply:
Oh, if that's the case, better to keep it like this. Thanks!